.circleci/workflows.yml

```diff
@@ -254,6 +254,8 @@ jobs:
     executor: vm
     environment:
       ARBITRUM_FOLDER: "docker/arbitrum"
+      DOCKER_BUILDKIT: 0
+      COMPOSE_DOCKER_CLI_BUILD: 0
```

Comment:
Just googled and wanted to try out, and it worked

Reply:
https://stackoverflow.com/a/66695181

Reply:
But do we know _why_ this works? And how this interacts with CircleCI's DLC (Docker Layer Caching)?

Reply:
With BuildKit, you don't need to pull the remote images before building since it caches each build layer in your image registry. Then, when you build the image, each layer is downloaded as needed during the build.

So i guess it will disable it for arbitrum e2e test

Reply:
Because this is disabling buildkit simply